assets/survey_themes/fruity_twentythree/surveylist/surveylist.scss

```diff
@@ -5,6 +5,10 @@ html, body {
 
 #outerframeContainer {
     min-height: 88%;
+
+    .main-content {
+        max-width: 750px;
+    }
```

Comment:
only the welcome container texts should be limited not all containers for questions as well